src/features/classification/security-controls/mixins.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../styles/variables';
+
+$scrollbar-width: 7px;
+
+@mixin BackgroundlessScrollbar {
```

Comment:
This mixin should be prefixed with `bdl-` if it's needed.

Reply:
reverted